### DISCUSSION TITLE
personal project: covid tracker

### DISCUSSION THREAD

**Discussion by Phillip-D-Shields** [vercel/next.js#17387] [Show and tell]
I made a nextjs app to share simple and unpoliticized data with my family and friends back home in Michigan. I am still learning how to use nextjs properly and would appreciate any advice or suggestions. I am especially curious if there is a better and/or cleaner way to use the `getServerSideProps` ?  I am hoping to do a refractor soon to turn most of the functions into helper functions, but other than that I am not experienced enough to know how to make this app even better. 

my repo is [here ](https://github.com/Phillip-D-Shields/next-material-c19-app)

the live app is [here](https://mi-us-int-c19.vercel.app), but i havent tweaked everything for mobile so bigger screen is better for. chartsjs charts are hidden on small viewports, and really they are the best part for me...

**Comment by lachlanjc** [comment]
Link to the site is broken :( (& on the GitHub repo too)
